nixos/modules/services/monitoring/loki.nix

```diff
@@ -97,7 +97,12 @@ in {
 
       serviceConfig = let
         conf = if cfg.configFile == null
-               then prettyJSON cfg.configuration
+               then
+                 # Config validation may fail when using extraFlag = [ "-config.expand-env=true" ].
+                 # To work around this, we simply skip it when extraFlags is not empty.
+                 if cfg.extraFlag == []
```

Comment:
This doesn't eval, I think it should be `extraFlags`

Reply:
oh whoops, sorry